content/en/blog/_posts/2020-01-17-Docs-Review-2019.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: blog
+title: "Reviewing 2019 in Docs"
+date: 2020-01-17
+slug: reviewing-2019-in-docs
+---
+
+**Author:** Zach Corleissen (Cloud Native Computing Foundation)
+
+Hi, folks! I'm one of the co-chairs for the Kubernetes documentation special interest group (SIG Docs). This blog post is a review of SIG Docs in 2019. Our contributors did amazing work last year, and I want to highlight their successes. 
+
+Although I review 2019 in this post, my goal is to point forward to 2020. I observe some trends in SIG Docs--some good, others troubling. I want to raise visibility before those challenges increase in severity.
+
+## The good
+
+There was much to celebrate in SIG Docs in 2019.
+
+Kubernetes docs started the year with three localizations in progress. By the end of the year, we ended with ten localizations available, four of which (Chinese, French, Japanese, Korean) are reasonably complete. The Korean and French teams deserve special mentions for their contributions to git best practices across all localizations (Korean team) and help bootstrapping other localizations (French team). 
```

Comment:
```suggestion
Kubernetes docs started the year with three localizations in progress. By the end of the year, we ended with ten localizations available, four of which (Chinese, French, Japanese, Korean) are reasonably complete. The Korean and French teams deserve special mentions for their contributions to git best practices across all localizations (Korean team) and help to bootstrap other localizations (French team).
```

Reply:
The trailing space needs to go, but "bootstrapping" seems appropriate here.